source/api/commands/task.md

```diff
@@ -176,6 +176,50 @@ module.exports = (on, config) => {
 }
 ```
 
+### Save a variable across non same-origin URL visits
```

Comment:
this works, ran the code, I will code a recipe just to point people and live tested example. Great stuff, @jennifer-shehane